index.js

```diff
@@ -93,6 +91,27 @@ function submitted(event) {
                     'event_label': info,
                 });
 
+                // Get global IP address
```

Comment:
This code should be fairly easy to understand but please let me know if it isn't.